File: CRM/src/app/shared/eHospital_DongNai_A_NSTL/NS_DM_QUANHUYEN.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { NS_DM_QUANHUYEN } from './NS_DM_QUANHUYEN.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class NS_DM_QUANHUYENService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'MaQuanHuyen', 'TenQuanHuyen', 'MaTinh', 'DangSuDung'];

    List: NS_DM_QUANHUYEN[] | undefined;
    ListFilter: NS_DM_QUANHUYEN[] | undefined;
    FormData!: NS_DM_QUANHUYEN;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "NS_DM_QUANHUYEN";
    }
}

File: Service_eHospital_DongNai_A_NSTL/Implement/NS_DM_QUANHUYENService.cs
namespace Service_eHospital_DongNai_A_NSTL.Implement
{
    public class NS_DM_QUANHUYENService : BaseService<NS_DM_QUANHUYEN, INS_DM_QUANHUYENRepository>
    , INS_DM_QUANHUYENService
    {
    private readonly INS_DM_QUANHUYENRepository _NS_DM_QUANHUYENRepository;
    public NS_DM_QUANHUYENService(INS_DM_QUANHUYENRepository NS_DM_QUANHUYENRepository) : base(NS_DM_QUANHUYENRepository)
    {
    _NS_DM_QUANHUYENRepository = NS_DM_QUANHUYENRepository;
    }
    }
    }

